tests/Predis/Command/Redis/TDigest/TDIGESTQUANTILE_Test.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Predis package.
+ *
+ * (c) 2009-2020 Daniele Alessandri
+ * (c) 2021-2023 Till Krüss
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Predis\Command\Redis\TDigest;
+
+use Predis\Command\Redis\PredisCommandTestCase;
+use Predis\Response\ServerException;
+
+class TDIGESTQUANTILE_Test extends PredisCommandTestCase
+{
+    /**
+     * {@inheritDoc}
+     */
+    protected function getExpectedCommand(): string
+    {
+        return TDIGESTQUANTILE::class;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function getExpectedId(): string
+    {
+        return 'TDIGESTQUANTILE';
+    }
+
+    /**
+     * @group disconnected
+     */
+    public function testParseResponse(): void
+    {
+        $this->assertSame(1, $this->getCommand()->parseResponse(1));
+    }
+
+    /**
+     * @group connected
+     * @return void
+     * @requiresRedisBfVersion >= 2.4.0
+     */
+    public function testquantile(): void
+    {
+        $redis = $this->getClient();
+
+        $redis->tdigestcreate('key', 1000);
+
+        $addResponse = $redis->tdigestadd('key', 1, 2, 2, 3, 3, 3, 4, 4, 4, 4, 5, 5, 5, 5, 5);
+        $quantileResponse = $redis->tdigestquantile('key', 0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0);
+
+        $this->assertEquals('OK', $addResponse);
+        $this->assertSame(['1', '2', '3', '3', '4', '4', '4', '5', '5', '5', '5'], $quantileResponse);
```

Comment:
Add one more assertion to test empty TDigest sketch:
`$redis->tdigestcreate('empty_key');`
`$this->assertSame(['nan', 'nan'], $redis->tdigestquantile('empty_key', 0.0, 0.1));`